File: WEKEE.FE/UI.HOME/src/scenes/Main/HDetail/dtos/cartBasicProductDto.ts
import { SpecificationAttributeDto } from "./specificationAttributeDto";

export interface CartBasicProductDto {
    name: string;
    trademark : string,
    fullDescription: string;
    shortDescription: string;
    hasUserAgreement: boolean;
    orderMinimumQuantity: number;
    orderMaximumQuantity: number;
    shipSeparately: boolean;
    isFreeShipping: boolean;
    backorderModeId: number;
    disableWishlistButton: boolean;
    wishlistNumber: number;
    markAsNew: boolean;
    markAsNewStartDateTimeUtc: string | null;
    markAsNewEndDateTimeUtc: string | null;
    specificationAttributeDtos : SpecificationAttributeDto[]
}

File: WEKEE.BE/Product.Domain/Shared/DataTransfer/ProductDTO/CartBasicProductDto.cs
using Product.Domain.Shared.DataTransfer.SpecificationAttributeDTO;
using System;
using System.Collections.Generic;
using System.Text;

namespace Product.Domain.Shared.DataTransfer.ProductDTO
{
    public class CartBasicProductDto
    {
        public string Name { get; set; }
        public string Trademark { get; set; }
        public string FullDescription { get; set; }
        public string ShortDescription { get; set; }
        public bool HasUserAgreement { get; set; }
        public int OrderMinimumQuantity { get; set; }
        public int OrderMaximumQuantity { get; set; }
        public bool ShipSeparately { get; set; }
        public bool IsFreeShipping { get; set; }
        public int BackorderModeId { get; set; }
        public bool DisableWishlistButton { get; set; }
        public int WishlistNumber { get; set; }
        public bool MarkAsNew { get; set; }
        public DateTime? MarkAsNewStartDateTimeUtc { get; set; }
        public DateTime? MarkAsNewEndDateTimeUtc { get; set; }
        public List<SpecificationAttributeDto> SpecificationAttributeDtos { get; set; }
    }
}
